src/QueryBuilder/QueryLoader/QueryLoader.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Card, Icon, Loading, Row, Headline } from '@folio/stripes/components';
+
+const QueryLoader = ({ title }) => {
```

Comment:
I believe QueryLoader can be incapsulated in QueryViewer module as well

Reply:
QueryLoader should be shared, because it will be required in QueryBuilderModal as well

Reply:
in modal you'll use table itself, no?

Reply:
Yea, your're right. Fixed